app/Http/Requests/MemberInviteRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class MemberInviteRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required',
+            'email' => 'required | email',
+            'role_id' => 'required | integer',
```

Comment:
@Babita3331 Please add a rule to check whether this `role_id` actually exists in some database table. For example, if the `role_id` is only up to 5, I may still be able to insert random value 434 and since you have not maintain foreign key constraints in migration, these won't throw error